test/e2e/stub.js

```diff
@@ -0,0 +1,22 @@
+const playwright = require(process.cwd());
+
+if (process.argv.length === 2) {
+  console.error("Usage stub.js <browser-types-space-separated>")
+  process.exit(1)
+}
+
+const browsers = process.argv.slice(2, process.argv.length);
+
+(async () => {
+  for (const browserType of browsers) {
+    const browser = await playwright[browserType].launch();
+    const context = await browser.newContext();
+    const page = await context.newPage();
+    await page.goto('http://whatsmyuseragent.org/');
```

Comment:
let's keep this hermetic to avoid extra requests to third-party websites:

```js
await page.evaluate(() => navigator.userAgent);
```

Reply:
makes fully sense, +1.